pages/common/psalm.md

```diff
@@ -0,0 +1,32 @@
+# psalm
+
+> A static analysis tool for finding errors in PHP applications.
+> More information: <https://psalm.dev>.
+
+- Generate a Psalm configuration:
+
+`psalm --init`
+
+- Analyse the current working directory:
+
+`psalm`
+
+- Analyse a specific directory or file:
+
+`psalm {{path/to/directory_or_file}}`
```

Comment:
we always use `file` first.
```suggestion
`psalm {{path/to/file_or_directory}}`
```

Reply:
Good spot, I think I originally wrote `path/to/directory`, and then added `or_file` to the end. But it sounds really odd that way. 👍🏻